### DISCUSSION TITLE
Understanding about nan values

### DISCUSSION THREAD

**Discussion by emcomit** [open-meteo/open-meteo#315] [Q&A]
Dear open-meteo team,
if on my self-hosted open-meteoapi installation, when I request for example the datas:
`v1/forecast?latitude=52.52&longitude=13.41&hourly=rain&daily=apparent_temperature_max,sunrise&current_weather=true&timezone=Europe%2FBerlin`

the api return me:

`{"latitude":52.52,"longitude":13.419998,"generationtime_ms":14.195084571838379,"utc_offset_seconds":7200,"timezone":"Europe/Berlin","timezone_abbreviation":"CEST","elevation":45.0,"current_weather":**{"temperature":nan,"windspeed":nan,"winddirection":nan,"weathercode":null**,"time":"2023-05-01T15:00"},"hourly_units":{"time":"iso8601","rain":"mm"},"hourly":{"time":["2023-05-01T00:00",.....,"2023-05-07T23:00"],"rain":[0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.10,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.10,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00,0.00]},"daily_units":{"time":"iso8601","apparent_temperature_max":"°C","sunrise":"iso8601"},"daily":{"time":["2023-05-01","2023-05-02","2023-05-03","2023-05-04","2023-05-05","2023-05-06","2023-05-07"],"apparent_temperature_max":[**null,null,null,null,null,null,null**],"sunrise":["2023-05-01T05:33","2023-05-02T05:31","2023-05-03T05:29","2023-05-04T05:27","2023-05-05T05:25","2023-05-06T05:23","2023-05-07T05:21"]}}`

probably I have some issue about the download of some weather models.
Which models (ERA, GEM, GFS, etc) are responsible for showing these data? Is available a documentation that explain that?
Thanks

**Comment by patrick-zippenfenig** [comment]
Hi, if you only downloaded the ECMWF IFS model, `apparent_temperature` wont be available, because is uses solar radiation. Depending on the region, you also have to download the GFS or ICON model.
